.circleci/config.yml

```diff
@@ -10,8 +10,18 @@ executors:
     resource_class: large
   appimage-executor:
     docker:
-      - image: openscad/appimage-x86_64-base:latest
+      - image: ${CIRCLE_PROJECT_USERNAME}/appimage-base:<< pipeline.git.revision >>
     resource_class: large
+  docker-x86_64:
+    machine:
+      image: ubuntu-2204:current
+      docker_layer_caching: true
+    resource_class: large
+  docker-aarch64:
```

Comment:
We tend to use arm64 for this arch, but I'm not familiar with conventions used elsewhere..